Disable poll support (introduce a syntax error into this file) due to
poor performance.
https://trac.macports.org/ticket/10637
--- src/trypoll.c.orig	2001-07-12 11:49:49.000000000 -0500
+++ src/trypoll.c	2023-03-05 19:55:12.000000000 -0600
@@ -18,3 +19,4 @@
 
   _exit(0);
 }
+x
